zzzz_modular_occulus/code/game/occulus-lore-fixes.dm

```diff
@@ -193,8 +193,6 @@
 	name = "Kampfer Roach"
 	desc = "A monstrous, dog-sized cockroach. These huge mutants can be found everywhere on the ship."
 
-/obj/item/weapon/tool_upgrade/augment/randomizer
```

Comment:
Is this removal intended? It would revert randomizer name, rarity, and give it the loot spawn tag back